src/components/vmd-appointment-card.component.scss

```
@@ -31,6 +31,26 @@
 .card {
   max-width: 80vw;
   margin: auto;
+  overflow: hidden;
+
+  .highlight-text {
+    display: block;
+    text-align: center;
+    font-weight: bold;
+    background-color: tint-color($primary, 80%);
+
+    color: $primary;
+    margin: 0px;
+    padding: 5px;
+  }
+
+  &.highlighted {
+  }
+  &:not(.highlighted) {
+    .highlight-text {
+      display: none;
+    }
+  }
```

Hello, pour simplifier la règle d'affichage: 
- par défaut on cache le texte
- si `highlighted` : on l'affiche
```suggestion
  .highlight-text {
    display: none;
    text-align: center;
    font-weight: bold;
    background-color: tint-color($primary, 80%);    
    color: $primary;
    margin: 0px;
    padding: 5px;
  }

  &.highlighted .highlight-text {
      display: block;     
  }
```